### DISCUSSION TITLE
我添加的知识库的文件一直是在排队中是怎么回事啊？

### DISCUSSION THREAD

**Discussion by xiahao90** [langgenius/dify#10617] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

我添加的知识库的文件一直是在排队中是怎么回事啊？
![image](https://github.com/user-attachments/assets/bd7817a3-0754-47d8-bf91-2d23a59e11b2)

**Comment by jianhuisu** [comment]
我参与讨论下哈 
有两个可能 
1 服务器没有启动 celery 消费队列任务
2 知识库中第一个任务没有完成 或者因为其它原因卡死. 导致后续的任务被阻塞

另外为啥你这些文档的创建时间都是相同的

**Reply by xiahao90** [reply]
好了，没有配置LOG_FILE是个巨坑，导致woker容器报错

**Comment by xiahao90** [comment, marked as answer]
doker-compoer启动的朋友，记得要在.env中配置这个才行，配置后restart没有效果，需要docker-compose up -d。有缘人搜到有用来个👍

LOG_FILE=/app/logs/server.log

**Reply by YoshuaBengio** [reply]
配了日志，还是排队，怎么解决

**Comment by emocat17** [comment]
可以看一下这个：[Dify 知识库文件上传一直处于“排队中”问题的解决方案](https://blog.csdn.net/qq_21648251/article/details/153478592?sharetype=blogdetail&sharerId=153478592&sharerefer=PC&sharesource=qq_21648251&spm=1011.2480.3001.8118)
